src/dataset-builder/ConceptSearch.test.ts

```diff
@@ -101,11 +101,7 @@ describe('ConceptSearch', () => {
     await user.click(await screen.findByLabelText(`add ${concept.id}`));
     await user.click(screen.getByLabelText(`open hierarchy ${concept.id}`));
     // Assert
-    expect(onOpenHierarchy).toHaveBeenCalledWith(
-      { id: concept.id, category: domainOption.category, root: concept },
-      [concept],
-      ''
-    );
+    expect(onOpenHierarchy).toHaveBeenCalledWith(domainOption, [concept], '', concept);
```

Comment:
why the extra `concept`, well onOpenHierarchy has an optional parameter `openedConcept`, which is the concept that is being opened. 

These are the arguments for onOpenHierarchy in this scenario
toHaveBeenCalledWith(domainOption: SnapshotBuilderDomainOption, cart: Concept[], searchText: string, openedConcept?: Concept)